services/spar/src/Spar/Scim.hs

```diff
@@ -1,8 +1,8 @@
 {-# LANGUAGE DuplicateRecordFields #-}
 {-# LANGUAGE OverloadedLists #-}
 {-# LANGUAGE RecordWildCards #-}
-{-# OPTIONS_GHC -Wno-missing-methods
-    -Wno-orphans #-}
+{-# OPTIONS_GHC -Wno-missing-methods #-}
```

Comment:
I wonder how that got in...?  :)

(I would have removed this and at least implemented the methods as `undefined`, but I guess that's not a great difference in terms of runtime behavior.)

Reply:
And why is the instance needed, but not defining any of its methods doesn't break? How has this been in prod for two years? :grimacing: 

Looking into it now.

Reply:
So it seems like we host the full Scim `Site`, including the `GroupSite`, but the implementation for that goes through the `GroupDB` class, which would fail any time one of these endpoints is called.

I just made a commit to only host the `UserSite` and `ConfigSite` by copying a bit of code from `Web.Scim.Server`. This means people accessing group endpoints would get a proper 404 without hitting undefined methods. How does that sound to you?

About to push that in a moment.